pkg/plugin/plugin.go

```diff
@@ -38,9 +35,16 @@ type issueMatcher interface {
 	MatchIssue(context.Context, string) (*validator.MatchResult, error)
 }
 
+// The JiraUIData comprises the data that will be displayed. At present, it exclusively includes the displayName and hint.
+type JiraUIData struct {
```

Comment:
Use jvspb.UIData directly instead of creating another struct?

Reply:
+1 - or alias it `type JiraUIData = jvspb.UIData` if you need it to be local to the package.